examples/README.MD

```diff
@@ -0,0 +1,353 @@
+## FL4Health Examples
+
+This folder contains a large collection of example implementations of the various methods and capabilities of the FL4Health library. There are several broad categories covered by these examples, including different state-of-the-art global and personalized FL (pFL) methods, aggregation strategies, pre-processing techniques and more.
```

Comment:
Maybe add a link to the main README to point the user to the descriptions of the different strategies so that we can get away with not defining them here

Reply:
By descriptions I mainly mean just an expansion of the acronyms

Reply:
Sure. Will add here as well.